rcldotnet/RCLdotnet.cs

```diff
@@ -199,8 +199,14 @@ public class RCLdotnet {
     public static Node CreateNode (string nodeName, string nodeNamespace) {
       IntPtr nodeHandle = IntPtr.Zero;
       int ret = RCLdotnetDelegates.native_rcl_create_node_handle (ref nodeHandle, nodeName, nodeNamespace);
-      Node node = new Node (nodeHandle);
-      return node;
+
+      if (ret != -1)
```

Comment:
This is a magic number, it should translate to a constant with a meaningful name.

Reply:
Ok. I added a `NODE_NAME_ERROR` both to C and C# code in order to remove magic numbers.